### ISSUE TITLE
Hydroenergy Hydro electric Dam causes game to crash when removing water

### ISSUE THREAD

**Issue by SEGPPMEJPTF** [GTNewHorizons/GT-New-Horizons-Modpack#12616]
### Your GTNH Discord Username

SEGPPMEJPTF (Max P)#5629

### Your Pack Version

2.2.9 MMC

### Your Server

SP

### Java Version

Java 8

### Type of Server

None

### Your Actions

![2023-02-16_12 27 35](https://user-images.githubusercontent.com/125594780/219455691-fadcbea7-eab2-4e6f-8839-7bd9eac8f4db.png)
TLDR at bottom

When attempting to use the "Hydro Dam 2086:17000" from the "Greg Tech" mod, the water in the dam does not render properly. The water is not visible and appears at ranges that are not expected. Additionally, when any part of the dam multiblock is broken, the light concrete appears to change to dark concrete until it is right clicked on, changing it back. When trying to access the multiblock while broken or trying to change the visualization of the water, via the three buttons in the top right when right clicked with a screwdriver, the game crashes stating that it is shutting down the internal server and then reports that a fatal error has occured. 
I've tried restarting the world multiple times and loading aromacore backups but the problem persists that whenever rendering or un-rendering the water for the Hydrodam a fatal error occurs.
![2023-02-16_12 24 04](https://user-images.githubusercontent.com/125594780/219455661-14d5ebbf-f9d0-4dba-80b8-0b76bd701d5a.png)
![2023-02-16_12 24 11](https://user-images.githubusercontent.com/125594780/219455669-625b47d1-234e-4a48-8fbf-52ac15c03d96.png)

TLDR; Hydrodam crashes game due to buggy visualization
![2023-02-16_12 25 10](https://user-images.githubusercontent.com/125594780/219455678-e15faf3b-9006-4213-817a-c847a22cada3.png)
![2023-02-16_12 25 26](https://user-images.githubusercontent.com/125594780/219455685-c267ade5-54bf-46a3-8aeb-7fb2afe35c69.png)
![2023-02-16_12 26 05](https://user-images.githubusercontent.com/125594780/219455688-22a3a082-fd7d-4e0f-9a06-ef193836c31e.png)
![2023-02-16_12 26 12](https://user-images.githubusercontent.com/125594780/219455689-e23ffede-ead4-4f2f-b668-39d9583e082e.png)

@SinTh0r4s 

### Crash Report

https://paste.ubuntu.com/p/B9gdp8mYrb/

[crash-2023-02-16_10.52.48-client.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/10759242/crash-2023-02-16_10.52.48-client.txt)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [x] I have asked other people and they confirm they also crash by following the exact steps I described above.
